### PR TITLE
Fix android VideoPlayer size calculation when keepRatio == true.

### PR DESCRIPTION
**Pull request by gin0606**
Cocos2dxVideoView has problem when `videoPlayer->setKeepAspectRatioEnabled(true);`
# reproduce step
## modify code
1. Please Go `UIVideoPlayerTest.cpp` -> `VideoPlayerTest::createVideo()`
2. modify code same as ↓this.

This code want to show video that fit screen.

``` cpp
void VideoPlayerTest::createVideo()
{
    auto centerPos = Vec2(_visibleRect.origin.x + _visibleRect.size.width / 2,_visibleRect.origin.y + _visibleRect.size.height /2);

    auto widgetSize = _widget->getContentSize();

    _videoPlayer = VideoPlayer::create();
    _videoPlayer->setPosition(centerPos);
    _videoPlayer->setAnchorPoint(Vec2::ANCHOR_MIDDLE);
    // I modify this line.
    _videoPlayer->setContentSize(Director::getInstance()->getVisibleSize());
    _videoPlayer->setKeepAspectRatioEnabled(true);

    _uiLayer->addChild(_videoPlayer);

    _videoPlayer->addEventListener(CC_CALLBACK_2(VideoPlayerTest::videoEventCallback, this));
}
 _videoPlayer->addEventListener(CC_CALLBACK_2(VideoPlayerTest::videoEventCallback, this));
```
## run android cpp-tests
1. Run `cpp-tests`.
2. Go to `Node::UI -> GUI Dynamic Create Test -> VideoPlayerTest`
3. Tap `Play resource video` label
4. problem reproduce
5. Tap `KeepRationSwitch`
6. Another problem reproduce

### CODE REVIEW

File: cocos/platform/android/java/src/org/cocos2dx/lib/Cocos2dxVideoView.java [hunk 1]
```diff
@@ -324,27 +324,18 @@ public void fixSize() {
...
-    			if ( mVideoWidth * height  > width * mVideoHeight ) {
-    				mVisibleWidth = width;
-    				mVisibleHeight = width * mVideoHeight / mVideoWidth;
-                } else if ( mVideoWidth * height  < width * mVideoHeight ) {
-                	mVisibleWidth = height * mVideoWidth / mVideoHeight;
-                	mVisibleHeight = height;
-                }
-    			mVisibleLeft = left + (width - mVisibleWidth) / 2;
-    			mVisibleTop = top + (height - mVisibleHeight) / 2;
-			} else {
```
Comment (WenhaiLin) [inline cocos/platform/android/java/src/org/cocos2dx/lib/Cocos2dxVideoView.java:343]: Why ignore this case?Did you test size calculation when keepRatio is false?

Comment (WenhaiLin) [inline cocos/platform/android/java/src/org/cocos2dx/lib/Cocos2dxVideoView.java:343]: I submit another pr for fix this issue at https://github.com/cocos2d/cocos2d-x/pull/7597

Comment (gin0606) [inline cocos/platform/android/java/src/org/cocos2dx/lib/Cocos2dxVideoView.java:343]: oops this case can not ignore.
I will fix it.

And I will restore 122L and 327L

Comment (WenhaiLin) [inline cocos/platform/android/java/src/org/cocos2dx/lib/Cocos2dxVideoView.java:343]: The 122L and 327L needed no restore.Your modification is right.

Comment (gin0606) [inline cocos/platform/android/java/src/org/cocos2dx/lib/Cocos2dxVideoView.java:343]: OK. I rebased cocos2d-x latest `v3` branch.

Comment (WenhaiLin) [inline cocos/platform/android/java/src/org/cocos2dx/lib/Cocos2dxVideoView.java:343]: I think the result of size calculation is correct.Please describe  in more detail.

Comment (gin0606) [inline cocos/platform/android/java/src/org/cocos2dx/lib/Cocos2dxVideoView.java:343]: You are right!
Already fixed issue on https://github.com/cocos2d/cocos2d-x/pull/7597 !

I will close this pr.
Thank you your review :bow:
